src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlUtil.cs

```diff
@@ -145,7 +145,7 @@ internal static Task CreateContinuationTaskWithState(Task task, object state, Ac
         )
         {
             task.ContinueWith(
-                tsk =>
+                (Task tsk, object state2) =>
```

Comment:
NIT: a better naming convention would be helpful here for state2.

Reply:
Any suggestions? It's the a state param which is just going to be cast to the correct type and it only needs to be named differently from the one in the parent scope to prevent confusion.

Reply:
that is fine.